### DISCUSSION TITLE
序盤から生成される冠扱いのベース装備が欲しい

### DISCUSSION THREAD

**Discussion by daradarach** [hengband/hengband#219] [Ideas]
冠につく各種エゴは序盤から非常に有用だが、冠扱いのベースアイテムの生成階層が深いため序盤では手に入らない。
生成階層の浅い冠扱いのベースアイテムが欲しい。
草冠でも花冠でも月桂冠でも羽根帽子とかでも。

**Comment by daradarach** [comment]
月桂冠
AC0(強化できない)
火属性の攻撃で燃える(永久的に失われる？劣化する？)

**Comment by daradarach** [comment]
ベースアイテム:髪飾り

**Comment by daradarach** [comment]
ベースアイテム:ヘアバンド

**Comment by gold-onion** [comment]
α11で鉄冠の生成階が40→15階になったようです

**Reply by daradarach** [reply]
それとは別でフレーバーっぽさのある冠のアイデアをDiscordで話してたのでそのメモです
